doc/classes/SpringArm3D.xml

```diff
@@ -28,6 +28,12 @@
 				Returns the spring arm's current length.
 			</description>
 		</method>
+		<method name="is_colliding" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns whether the SpringArm3D's ray or shape is intersecting with any colliders, and thus, if the spring arm's current length is less than its maximum length.
```

Comment:
```suggestion
				Returns whether the [SpringArm3D]'s ray or shape is intersecting with any colliders, and thus, if the spring arm's current length is less than its maximum length.
```

Reply:
Unnecessary here I'd say, it's the same class, and isn't done anywhere else in the file

Reply:
Hm okay, it would be best to change these as a sweeping change then. Same class is displayed with bold letters, making it more readable.

Reply:
Agreed, but there are seven other cases so it feels like it'd be confusing 🙂

Reply:
> Hm okay, it would be best to change these as a more sweeping change then. Same class is displayed with bold letters, making it more readable.

I wouldn't mind adding it to the entire file. Should I proceed?^^

Reply:
I'd say to leave it now and fix in a separate PR